packages/react/src/Suspense.tsx

```diff
@@ -16,16 +16,13 @@ export interface SuspenseProps extends ReactSuspenseProps {
  * @see {@link https://suspensive.org/docs/react/Suspense Suspensive Docs}
  */
 export const Suspense = Object.assign(
-  ({ clientOnly, children, fallback }: SuspenseProps) => {
+  ({ clientOnly, fallback, ...rest }: SuspenseProps) => {
```

Comment:
name could be contained in rest object because React.SuspenseProps have name prop

```tsx
interface SuspenseProps {
  children?: ReactNode | undefined;

  /** A fallback react tree to show when a Suspense child (like React.lazy) suspends */
  fallback?: ReactNode;

  /**
  * A name for this Suspense boundary for instrumentation purposes.
  * The name will help identify this boundary in React DevTools.
  */
  name?: string | undefined;
}
```